storyblok/components.json

```diff
@@ -1617,6 +1614,149 @@
       "all_presets": [],
       "preset_id": null,
       "real_name": "title_paragraph_item"
+    },
+    {
+      "name": "card_checklist_bullet_point_block",
+      "display_name": "",
+      "schema": {
+        "color": {
+          "type": "custom",
+          "field_type": "hedvig-limited-color-picker",
+          "options": [],
+          "pos": 0
+        },
+        "bullet_points_position": {
+          "type": "option",
+          "options": [
+            {
+              "value": "left",
+              "name": "Left"
+            },
+            {
+              "value": "center",
+              "name": "Center"
+            },
+            {
+              "value": "right",
+              "name": "Right"
+            }
+          ],
+          "default_value": "left",
+          "pos": 1
+        },
+        "enforce_size": {
+          "type": "boolean",
+          "pos": 2,
+          "default_value": "true",
+          "description": "By unchecking this, I hereby promise that the images i upload are of exacly the same size"
+        },
+        "bullet_points": {
+          "type": "bloks",
+          "maximum": "6",
+          "restrict_components": true,
+          "component_whitelist": ["card_checklist_bullet_point_item"],
+          "required": true,
+          "pos": 3,
+          "translatable": false
+        }
+      },
+      "image": null,
+      "preview_field": null,
+      "is_root": false,
+      "is_nestable": true,
+      "all_presets": [],
+      "preset_id": null,
+      "real_name": "card_checklist_bullet_point_block"
+    },
+    {
+      "name": "card_checklist_bullet_point_item",
+      "display_name": "",
+      "schema": {
+        "image": {
+          "type": "image",
+          "pos": 0,
+          "required": true,
+          "display_name": "",
+          "regex": "",
+          "image_crop": false,
+          "image_width": "375",
+          "image_height": "240"
+        },
+        "title": {
+          "type": "text",
+          "required": true,
+          "default_value": "",
+          "pos": 1,
+          "translatable": true
+        },
+        "paragraph": {
+          "type": "custom",
+          "default_value": "",
+          "required": true,
+          "pos": 2,
+          "field_type": "markdown-html",
+          "options": [],
+          "translatable": true
+        },
+        "check_list": {
+          "type": "options",
+          "pos": 3,
+          "translatable": true,
+          "options": [
```

Comment:
clever! However.. any particular reason for not making this a free-text field?

Reply:
i mean this will a) never be possible to translate and b) impossible for marketing to change if they want to reuse the block for anything else

Reply:
It's intentional that it can't be reused, it's dedicated to this particular use case. But maybe it would be better to add checklist support to image-bullet-point-item's free text instead?

Reply:
Great idea! Much better to have it as separate blocks